tests/dtypes/test_geometry.py

```diff
@@ -287,3 +277,46 @@ def test_loading(tmp_path: Path, rc: RawConfig):
     create_tabular_manifest(tmp_path / 'manifest.csv', table)
     manifest = load_manifest(rc, tmp_path / 'manifest.csv')
     assert manifest == table
+
+
+def test_geometry_params_with_srid_without_srid(
+    rc: RawConfig,
+    postgresql: str,
+    request: FixtureRequest,
+):
+    context = bootstrap_manifest(rc, f'''
+        d | r | b | m | property                | type           | access
+        backends/postgres/dtypes/geometry/srid  |                | 
+          |   |   | Point                       |                | 
+          |   |   |   | point                   | geometry       | open
+          |   |   | PointLKS94                  |                | 
+          |   |   |   | point                   | geometry(3346) | open
+          |   |   | PointWGS84                  |                | 
+          |   |   |   | point                   | geometry(4326) | open
+    ''', backend=postgresql, request=request)
+
+    ns: str = 'backends/postgres/dtypes/geometry/srid'
+    model: str = f'{ns}/PointLKS94'
+    store: Store = context.get('store')
+    backend = cast(PostgreSQL, store.backends['default'])
+    coordinates = cast(sa.Column, backend.tables[model].columns['point'])
+    assert coordinates.type.srid == 3346
+    assert coordinates.type.geometry_type == 'GEOMETRY'
+
+    app = create_test_client(context)
+    app.authmodel(model, [
+        'insert',
+        'getall',
+    ])
+
+    resp = app.post(f'/{model}', json={
+        'point': 'POINT(582710 6061887)',
+    })
+    assert resp.status_code == 201
+
+    resp = app.post(f'/{model}', json={
+        'point': 'POINT(582710 6061887)',
+    })
+    assert resp.status_code == 201
```

Comment:
Why you need to add exactly same thing, two times?

Reply:
Missed that.